packages/suite-desktop-core/src/app.ts

```diff
@@ -33,6 +33,8 @@ const createMainWindow = (winBounds: WinBounds) => {
         height: winBounds.height,
         minWidth: MIN_WIDTH,
         minHeight: MIN_HEIGHT,
+        titleBarStyle: 'hidden',
```

Comment:
according to docs, it is also available on windows

Reply:
there's no titleBar on Windows, unfortunately
![image](https://github.com/user-attachments/assets/05afe485-9ce5-4bd9-849f-dd12d62dc49d)

Reply:
It should be fixed now